nss-day-cohort-47/reactive-nutshell-palm-beach-pilots
# Add the grid and clickable grid items

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
- [x ] New feature (non-breaking change which adds functionality)
# Testing Instructions
Should display landing screen on grid and allow user to click item in grid to navigate to it
# Checklist:
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
